packages/core/src/reporters/clear-text-score-table.ts

```diff
@@ -59,12 +59,16 @@ class Column {
 
 class MutationScoreColumn extends Column {
   constructor(rows: MetricsResult, private readonly thresholds: MutationScoreThresholds) {
-    super('% score', (row) => row.metrics.mutationScore.toFixed(2), rows);
+    super('% score', (row) => (isNaN(row.metrics.mutationScore) ? 'n/a' : row.metrics.mutationScore.toFixed(2)), rows);
   }
   protected color(metricsResult: MetricsResult) {
-    if (metricsResult.metrics.mutationScore >= this.thresholds.high) {
+    const { mutationScore: score } = metricsResult.metrics;
+
+    if (isNaN(score)) return chalk.grey;
```

Comment:
Nitpicking: I think it's fine to use `if` without `else`, but I would like it to be consistent in this function. Also I think `if` should always use code blocks: `{ }`